crates/providers/Cargo.toml

```diff
@@ -11,11 +11,7 @@ opt-level = "z"
 lto = true
 
 [workspace]
-members = [
-    "http-server",
-    "kv-redis",
-    "nats",
-]
+members = ["http-server", "kv-redis", "nats"]
```

Comment:
This seems out of scope

Reply:
Same as above

Reply:
This will introduce merge conflicts on each change, e.g. if two providers are added in separate PRs, whereas that would not happen with the existing approach